### PR TITLE
Desktop styled bar

### PR DESCRIPTION
**Pull request by m4dz**

### CODE REVIEW

**Comment by m4dz** [comment]
@GoOz, I've got an issue when including SVG icons from the sprite. I can't figure why. If you can take a look at it, it'll be a great appreciated help ❤️

**Comment by GoOz** [comment]
@m4dz So far I can't tell why those SVGs aren't displayed. Did you get any luck since?

**Comment by m4dz** [comment]
@GoOz unfortunaltely: nope. This is my mission for the day 💣

**Comment by GoOz** [comment]
@m4dz Feel free to poke me if you need a new eye on this 😄

File: src/styles/bar.css [hunk 1]
```diff
@@ -5,7 +5,22 @@ body {
...
 
+
 [role=application] {
   height: initial;
   flex: 1 100vh;
 }
+
+
+[role=banner] {
+  --silver: #d6d8da;
```
Comment (CPatchane) [inline src/styles/bar.css]: CSS variables seems to be not very supported ([only in very recent browser](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_variables)), are they necessary here? /poke GoOz

Comment (GoOz) [inline src/styles/bar.css]: @CPatchane Yes, I pointed that out too on the mobile branch as well.

Comment (m4dz) [inline src/styles/bar.css]: We normally enable a fallback using autoprefixer, but it seems to not work here, which is a little bit annoying. I'll investigate, thanks.

Comment (GoOz) [inline src/styles/bar.css]: @m4dz Are you sure autoprefixer supports custom properties? I tried using CSSNext syntax on a project once and I had to use the postcss plugin **cssnext** to make it work. 
CSSNext plugin brings autoprefixer with it so i'm not sure which does what but here's what I know. 👅

Comment (m4dz) [inline src/styles/bar.css]: @GoOz you're right, my mistake, we'll pass through a postcss plugin to handle them 😉

**Comment by m4dz** [comment]
As I'm working on the optimizations right now on another branch (css variables, build chain, mq packing, etc), can we consider this one as valid and merge it? So I can reassemble my branches asap. Thanks!

**Comment by GoOz** [comment]
👍

**Pull request merged**